scg-system/src/main/java/io/telicent/core/SmartCacheGraph.java

```diff
@@ -52,6 +64,40 @@ public static FusekiServer.Builder smartCacheGraphBuilder() {
      * Builder for a Fuseki server configured for SmartCacheGraph
      */
     public static FusekiServer construct(String... args) {
+        YAMLConfigParser ycp = new YAMLConfigParser();
+        RDFConfigGenerator rcg = new RDFConfigGenerator();
+        String configPath;
+        String pattern = ".*\\.(yaml|yml)$";
+        Pattern regex = Pattern.compile(pattern);
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equalsIgnoreCase("--conf") && i + 1 < args.length ) {
```

Comment:
I haven't considered that there can be more than one config, I'm testing it now. The files do seem to be getting translated to RDF, but I can't query their services. I'm not sure if more than one config is allowed

Reply:
It looks like this would get logged as a warning in Jena and then it would just replace the first config value with the next one https://github.com/apache/jena/blob/c7eaf83cf328496cb4f8a24b42a483199ac216c0/jena-cmds/src/main/java/org/apache/jena/cmd/CmdLineArgs.java#L77

Reply:
I'm not getting that warning when testing the construct method. But the first configs do seem to be getting replaced or something else is happening to them, because I can make requests to the services defined in the last config in the list, but not to any from the configs before it. That happens regardless of whether either of the configs is in yaml or rdf, which would make sense considering that the yaml ones get replaced with rdf files before being passed to Fuseki.